nickytonline/dotca-bot
# chore: removed hard-coded base URL in guest info Edge function

## Description

Removes a hard-coded base URL when no guest is found in AirTable. As well, the Edge function is now named `guest-info`.

## Test Plan

1. Navigate to https://deploy-preview-5--iamdeveloperdotlive.netlify.app/guest-info
2. It should redirect to https://deploy-preview-5--iamdeveloperdotlive.netlify.app/background.html
3. Notice in the redirect that it no longer has a base URL. It's just `/background.html`.

![CleanShot 2022-11-07 at 22 40 44](https://user-images.githubusercontent.com/833231/200469920-6ae62ebd-c050-4c5e-9c95-8fb9051c0bf4.png)

# Relates Issues, PRs

Relates to #2
Relates to #3
